etc/tendrl/node-agent/node-agent.conf.yaml.sample

```diff
@@ -5,3 +5,4 @@ tendrl_ansible_exec_file: $HOME/.tendrl/node-agent/ansible_exec
 log_cfg_path: /etc/tendrl/node-agent/node-agent_logging.yaml
 log_level: DEBUG
 package_source_type: rpm
+logging_socket_path: "/tmp/node-agent/.node_agent.sock"
```

Comment:
I feel its better to have the socket file at "/var/run/tendrl/node-agent/.node_agent.sock" as /tmp is a bit vulnerable, user might clear /tmp directory.
@nthomas-redhat @shtripat please comment